app.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__license__ = """
+This file is part of OpenModelica.
+Copyright (c) 1998-CurrentYear, Open Source Modelica Consortium (OSMC),
+c/o Linköpings universitet, Department of Computer and Information Science,
+SE-58183 Linköping, Sweden.
+
+All rights reserved.
+
+THIS PROGRAM IS PROVIDED UNDER THE TERMS OF GPL VERSION 3 LICENSE OR
+THIS OSMC PUBLIC LICENSE (OSMC-PL) VERSION 1.2.
+ANY USE, REPRODUCTION OR DISTRIBUTION OF THIS PROGRAM CONSTITUTES
+RECIPIENT'S ACCEPTANCE OF THE OSMC PUBLIC LICENSE OR THE GPL VERSION 3,
+ACCORDING TO RECIPIENTS CHOICE.
+
+The OpenModelica software and the Open Source Modelica
+Consortium (OSMC) Public License (OSMC-PL) are obtained
+from OSMC, either from the above address,
+from the URLs: http://www.ida.liu.se/projects/OpenModelica or
+http://www.openmodelica.org, and in the OpenModelica distribution.
+GNU version 3 is obtained from: http://www.gnu.org/copyleft/gpl.html.
+
+This program is distributed WITHOUT ANY WARRANTY; without
+even the implied warranty of  MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE, EXCEPT AS EXPRESSLY SET FORTH
+IN THE BY RECIPIENT SELECTED SUBSIDIARY LICENSE CONDITIONS OF OSMC-PL.
+
+See the full OSMC Public License conditions for more details.
+"""
+
+import logging
+from flask import Flask, Blueprint
+import api
+import settings
+
+log = logging.getLogger(__name__)
+
+app = Flask(__name__)
+
+def configure_app():
+  logging.basicConfig(level=logging.DEBUG)

```

Comment:
It's ok to have debug level logging enabled for development purposes, but this should not be turned on in production by default. Could you please document how to set the logging level to ERROR in the README? It should be possible to do this via external application configuration, set via environment variables as per [12 factor app - config guidelines](https://12factor.net/config).